warehouse/jobs/runner.go

```diff
@@ -45,7 +45,7 @@ func (a *AsyncJobWhT) getTableNamesBy(sourceID, destinationID, jobRunID, taskRun
 	a.logger.Infof("[WH-Jobs]: Extracting table names for the job run id %s", jobRunID)
 	var tableNames []string
 	var err error
-	query := fmt.Sprintf(`SELECT id from %s where metadata->>'%s'=$1 and metadata->>'%s'=$2 and metadata in (SELECT metadata FROM wh_uploads where source_id=$3 and destination_id=$4)`, warehouseutils.WarehouseUploadsTable, "source_job_run_id", "source_task_run_id")
+	query := fmt.Sprintf(`SELECT id from %s where metadata->>'%s'=$1 and metadata->>'%s'=$2 and metadata in (SELECT metadata FROM %s where source_id=$3 and destination_id=$4)`, warehouseutils.WarehouseUploadsTable, "source_job_run_id", "source_task_run_id", warehouseutils.WarehouseUploadsTable)
```

Comment:
[minor] I believe it would be more readable this way
```suggestion
	query :=`SELECT id from`+warehouseutils.WarehouseUploadsTable+`where metadata->>'source_job_run_id'=$1 and metadata->>'source_task_run_id'=$2 and metadata in (SELECT metadata FROM ` + warehouseutils.WarehouseUploadsTable + ` where source_id=$3 and destination_id=$4)`
```

Reply:
done